modules/javafx.controls/src/test/java/test/javafx/scene/control/SplitMenuButtonTest.java

```diff
@@ -27,12 +27,14 @@
 
 import javafx.scene.control.MenuItem;
 import javafx.scene.control.SplitMenuButton;
+import javafx.scene.shape.Rectangle;
 import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
```

Comment:
Since you're modifying this file significantly, you could also use the JUnit5 imports.

Reply:
I think that would be out of scope for this PR.

We are considering updating tests to junit5 in bulk at some point, but I suspect it's a fairly low priority, unless we need to address something like https://bugs.openjdk.org/browse/JDK-8328629

Reply:
Well, you _do_ modify this file significantly, so it's not entirely out of scope. I've been converting files over to JUnit5 in the past as part of significant modifications, I think it's a valid path to get us to eventually move completely to JUnit5 (because as you say, bulk conversion isn't probably happening any time soon).

Reply:
I do that too, typically.  But here I just copied some code from a sibling test.

If we are converting, then we also should convert MenuButtonTest as well, and now we are widening the scope.  There is no gain in going with junit5 with this test specifically, so I don't really want to do it right now, I would rather do it as a part of a bulk change.

Do you want me to convert this test?

Reply:
created https://bugs.openjdk.org/browse/JDK-8338468

Reply:
It's okay, I'd just like to see it get done _eventually_.